alphaevolve_gs_engine/src/alphaevolve_gs_engine/services/validation/syntactic_validator.py

```
@@ -0,0 +1,207 @@
+"""
+syntactic_validator.py
+
+This module defines the SyntacticValidator, responsible for checking
+the syntactic correctness of policy code (e.g., Rego).
+
+Classes:
+    SyntacticValidator: Validates policy syntax.
+"""
+
+from typing import Tuple, Dict, Any, Optional
```

Ruff (0.11.9)

`typing.Dict` imported but unused

Remove unused import

(F401)